x/emissions/keeper/actor_penalties_test.go

```diff
@@ -78,6 +115,41 @@ func (s *KeeperTestSuite) TestApplyLivenessPenaltyToForecaster() {
 	s.Require().Equal(newScore, scoreFromStore)
 }
 
+// nolint: exhaustruct
+func (s *KeeperTestSuite) TestApplyLivenessPenaltyToActiveForecaster() {
+	ctx := s.ctx
+	keeper := s.emissionsKeeper
+
+	givenTopic := types.Topic{
+		Id:                  uint64(1),
+		MeritSortitionAlpha: alloraMath.MustNewDecFromString("0.1"),
+		EpochLastEnded:      105,
+		EpochLength:         10,
+		GroundTruthLag:      5,
```

Comment:
gt_lag needs to be equal or bigger than epochLength, not smaller. 
See https://github.com/allora-network/allora-chain/blob/6b2e5304c809773f84a6155ce091fe5fb8b02f6a/x/emissions/types/validations.go#L375

Separately, we should tetst with GTLag >> EpochLength which is where most problems will arise, since this leaves several rounds for active to overwrite inactive and so on.

Reply:
+1 here